core/modules/server/server.js

```diff
@@ -156,14 +155,15 @@ Server.prototype.isAuthorized = function(authorizationType,username) {
 	return principals.indexOf("(anon)") !== -1 || (username && (principals.indexOf("(authenticated)") !== -1 || principals.indexOf(username) !== -1));
 }
 
-Server.prototype.requestHandler = function(request,response) {
+Server.prototype.requestHandler = function(request,response,options) {
 	// Compose the state object
 	var self = this;
 	var state = {};
-	state.wiki = self.wiki;
+	state.wiki = (options?options.wiki:false) || self.wiki;
```

Comment:
We use whitespace in the ternary conditional operator: `(options ? options.wiki : false)`

Reply:
Actually, it would be more in line with existing code to insert a line `options = options || {}` at the top of the function, rather than guarding against each access.

Reply:
> Actually, it would be more in line with existing code to insert a line `options = options || {}` at the top of the function, rather than guarding against each access.

I considered this, but in the normal use case `options` is always undefined, and I thought this would prevent an extra object from being created but never used. I don't know if this is important or if I'm just over thinking it.

Reply:
We should be consistent with existing code.

Reply:
Ok, done.